src/document.js

```diff
@@ -171,7 +171,7 @@ const documentsMain = {
 				+ '</form>'
 
 			// iframe that contains the Collabora Online Viewer
-			const frame = '<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe_viewer" allowfullscreen allow="clipboard-read *; clipboard-write *" nonce="' + btoa(getRequestToken()) + '" style="width:100%;height:100%;position:absolute;"/>'
+			const frame = '<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe_viewer" allowfullscreen allow="clipboard-read *; clipboard-write *" nonce="' + btoa(getRequestToken()) + '" style="width:100%;height:100%;position:absolute;" title="Collabora Online Viewer"/>'
```

Comment:
it should be some "PRODUCT NAME" variable string I think

Reply:
@juliushaertl could you help here?

Reply:
https://github.com/nextcloud/richdocuments/blob/612379f2df0efc0002e8955538367c4b1eed0a02/src/view/Office.vue#L203 should work here as well

Reply:
@juliushaertl i am not quite sure how things work in .vue files and i am confused how to load product name in [src/document.js]. can you help me here ?

Reply:
is it like 
`<iframe data-cy="coolframe" id="loleafletframe" name="loleafletframe" nonce="' + btoa(getRequestToken()) + '" scrolling="no" allowfullscreen allow="clipboard-read *; clipboard-write *" style="width:100%;height:100%;position:absolute;" title="{productName}"/>`

Reply:
oh i see i think i can import the
 `import { getCapabilities } from './services/capabilities.ts' `
 
 and use `getCapabilities().productName` as an iframe title